sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationClientTestBase.java

```diff
@@ -286,7 +286,7 @@ void listConfigurationSettingsSelectFieldsRunner(BiFunction<List<ConfigurationSe
         tags.put("tag2", "value2");
 
         final SettingSelector selector = new SettingSelector()
-            .setLabelFilter("*-second*")
+            .setLabelFilter("my-second*")
```

Comment:
Maybe we can turn it into a negative test case

Reply:
if in the future, the service team start to support it again. Then we need to update the negative test case again?

Reply:
This would be a service API change. So, I hope the API version would be ramped and we'd expect the old API version to continue throwing the 400.

Reply:
make sense to me. I will add the negative test case